webapp-django/crashstats/crashstats/static/crashstats/js/socorro/report.js

```diff
@@ -18,7 +18,8 @@ $(document).ready(function() {
           $('#telemetryenvironment-json').JSONView(jsonData);
         } catch (ex) {
           console.warn('The data in the #telemetryenvironment-json dataset is not valid JSON');
-          container.append($('<p>Invalid JSON in TelemetryEnvironment</p>'));
+          container.append($('<p>Error when parsing JSON. Showing raw value:</p>'));
+          container.append($(document.createTextNode(jsonData)));
```

Comment:
The `$` wrapper isn't necessary, as `$.append` can take DOM nodes as arguments directly.

Reply:
Oo--good to know. I'll nix it.